### DISCUSSION TITLE
When to expect a release of spring boot starter

### DISCUSSION THREAD

**Discussion by ajdergute** [Unleash/unleash#2533] [General]
It is great that a spring boot starter is migrated. Do you have plans to release them?

**Comment by thomasheartman** [comment]
Great question! I think @praveenpg was looking into that just recently, so it shouldn't be too long 😄

**Comment by praveenpg** [comment]
@thomasheartman As I had mentioned earlier, I am not sure how do I make a release from the Unleash repo. I have done from my personal repo.
